components/integration/base/delete-snapshots.yaml

```diff
@@ -0,0 +1,42 @@
+#kind: Role
+#apiVersion: authorization.openshift.io/v1
+#metadata:
+#  name: delete-snapshots
+#rules:
+#  - apiGroups:
+#    - "appstudio.redhat.com"
+#    resources:
+#      - "snapshots"
+#    verbs:
+#      - "delete"
+#      - "deletecollection"
+---
+kind: RoleBinding
+#apiVersion: authorization.openshift.io/v1
+apiVersion: authorization.openshift.io/v1
+metadata:
+  name: delete-snapshots
+  namespace: integration-service
+subjects:
```

Comment:
instead of listing users, can you use a github group?

I see this group: https://github.com/orgs/redhat-appstudio/teams/integration-team which is missing people but I could help you to add people and we could use that one instead.

Reply:
Since that link gives a 404, we have a group for our team here: https://github.com/orgs/redhat-appstudio/teams/test-team which we can rename, if others agree

Reply:
The other gives a 404 because it is a private group and only member can see it. I add all the integration team member if you want or even better, I can grant maintainer role to someone in integration team who will be able to add other members.

Reply:
Or simply use test-team if this team already represents the integration team.

Reply:
test-team does have some people who aren't on the integration team, so we'll have to add or remove to/from one of the lists.  However, Kruno is the only maintainer of test-team and he's on PTO for the time being, so in the interest of speed @hugares can you add me as a maintainer to the integration-team group?  I can take care of the rest from there.

Reply:
@14rcole Done, you are a maintainer of https://github.com/orgs/redhat-appstudio/teams/integration-team